src/app/pages/create-wallet/create-wallet.tsx

```diff
@@ -1,7 +1,158 @@
-import { Component, h } from 'preact';
+import { h, Component } from 'preact';
+import { route } from 'preact-router';
+import Button from 'preact-material-components/Button';
+import LayoutGrid from 'preact-material-components/LayoutGrid';
+import 'preact-material-components/Card/style.css';
+import './create-wallet.scss';
+import Card from 'preact-material-components/Card';
 
 export class CreateWalletPage extends Component {
```

Comment:
if you use component state try to add an interface for the state and use generic types to specify it. In your case:

``` typescript
interface IState {
    words: string[],
    revealed: boolean
}

export class CreateWalletPage extends Component<{}, IState> { ... }
```

this way you enforce state type.